matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/typing/DefaultTypingService.kt

```diff
@@ -44,35 +43,33 @@ internal class DefaultTypingService @AssistedInject constructor(
         fun create(roomId: String): DefaultTypingService
     }
 
-    private var currentTask: Cancelable? = null
-    private var currentAutoStopTask: Cancelable? = null
+    private var job = Job()
```

Comment:
I think this should be a `CoroutineScope` instead of `Job`.

Reply:
So you would replace `private var job = Job()` by `private var coroutineScope = CoroutineScope(Job())` ?

Reply:
`private var coroutineScope = CoroutineScope()`

`Job` is added by default.

Reply:
The constructor of `CoroutineScope` requires a context

Reply:
Hmm, maybe I'm using a different version.
Well, it's the same either way.